bin/codecov.sh

```diff
@@ -100,19 +99,23 @@ if [[ -n ${FAILED_TESTS:-} ]]; then
   exit 1
 fi
 
+PKG_CHECK_ARGS=(--bucket='' )
+if [[ -n "${CIRCLE_BUILD_NUM:-}" ]]; then
+  if [[ -z "${CIRCLE_PR_NUMBER:-}" ]]; then
+    TMP_SA_JSON=$(mktemp /tmp/XXXXX.json)
+    ENCRYPTED_SA_JSON="${ROOTDIR}/.circleci/accounts/istio-circle-ci.gcp.serviceaccount"
+    openssl aes-256-cbc -d -in "${ENCRYPTED_SA_JSON}" -out "${TMP_SA_JSON}" -k "${GCS_BUCKET_TOKEN}"
```

Comment:
Commends ? Is this an env set for all the builds ? Will all builds have access to this ?

Reply:
it is only on postsubmit, but I think so. Unless you know of a way to filter it.